lnrpc/rpc.proto

```diff
@@ -1741,6 +1742,13 @@ message QueryRoutesRequest {
     A list of directed node pairs that will be ignored during path finding.
     */
     repeated NodePair ignored_pairs = 10;
+
+    /** 
+    An optional maximum total time lock for the route. This should not exceed
+    lnd's `--max-cltv-expiry` setting. If zero, then the value of
```

Comment:
So maybe for queryroutes, we should allow exceeding the max? Or only if the source pubkey is set and not equal to our own pubkey?

Reply:
Fixed.